### PR TITLE
[#275] Electron app and pyscript connected for power source component

### LINKED ISSUE #275: Bridge the gap between power souce py script and electron code
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current status?**
The individual codes work fine, but they must be brided together

* **What is the expected behaviour?**
The two components should work together 

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #275 

* **Please check if the PR fulfils these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [x] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
- Connected the electron app with python script for the power source component

* **Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ class App extends Component {
   }
 
   componentDidMount() {
+    // ipcRenderer.on('DEBUG', (event, args) => {
```
Comment (CloudyPadmal) [inline src/App.js]: If we don't need this, can remove the whole block

File: src/background_tasks/linker.html [hunk 2]
```diff
@@ -22,6 +22,7 @@
 		});
 
 		pyshell.on('message', function(results) {
+			// ipcRenderer.send('DEBUG', results);
```
Comment (CloudyPadmal) [inline src/background_tasks/linker.html]: Same here

**Comment by AakashMallik** [comment]
@CloudyPadmal Actually I left that for debug purpose if needed in near future while developing other components. I'll remove that anyway. I think it won't be needed as of now.

**Comment by AakashMallik** [comment]
@CloudyPadmal Done

**Pull request merged**